keyboards/bastardkb/charybdis/3x6/keymaps/via/config.h

```diff
@@ -21,8 +21,10 @@
 #    define DYNAMIC_KEYMAP_LAYER_COUNT 4
```

Comment:
this block could be removed too.  4 is the default.

Reply:
Done, and applied in the 4x6 PR as well.